postcss.json

```diff
@@ -0,0 +1,12 @@
+{
+  "autoprefixer": {
+    "browsers": [
+      "last 2 versions",
+      "safari >= 7",
+      "ie >= 9",
+      "ff >= 20",
+      "ios >=7",
+      "android >=4"
+    ]
+  }
+}
```

Comment:
@iondrimba what do you think about don't uses `postcss.json` and use a `browserslist` config? Example:

```
# browserslist
# Browsers that we support

Last 2 versions
Safari >= 7
ie >= 9
FF >= 20
iOS >=7
Android >=4
```

Whitout any extension, like documentation suggest https://github.com/ai/browserslist#browserslist.

This is more clean, what do you think?